code/modules/telesci/gps.dm

```diff
@@ -125,6 +135,10 @@ var/list/SPS_list = list()
 	if(href_list["toggle_refresh"])
 		autorefreshing = !autorefreshing
 		return TRUE
+		if(emped)
```

Comment:
This is just wrong: it will never be executed because it's right after a return. Copy-paste accident?